### PR TITLE
Update gameview.lua - fix bugs

### PR DESCRIPTION
**Pull request by neilswann80**
Allows first square to be clicked on at the cost of cycling back through clues in a loop.

Exits with a screen refresh

### CODE REVIEW

**Comment by roygbyte** [comment]
Thanks for the patch. I will test it soon.

File: gameview.lua [hunk 1]
```diff
@@ -159,6 +159,9 @@ function GameView:rightChar()
 end
 
 function GameView:leftChar()
+   if ( self.active_row_num == 1 and self.active_col_num == 1 ) then
```
Comment (neilswann80) [inline gameview.lua:162]: Prevents changing to previous clue when clicking on square (1,1)

File: gameview.lua
Comment (roygbyte) [inline gameview.lua:1]: I can confirm this avoids the bug for me too, but I'm unsure that it "fixes" the bug. Wouldn't this has a side effect of not allowing the `leftChar` method to move the selected grid item backwards, wrapping around to the last grid item?

File: gameview.lua [hunk 2]
```diff
@@ -160,10 +160,13 @@ end
 
 function GameView:leftChar()
    if (isTap == 1 and self.active_row_num == 1 and self.active_col_num == 1) then
+      isTap = 0
       return
    elseif (isTap == 1 and self.puzzle.size.cols > 15 and self.active_row_num < 3 and self.active_col_num == 1) then
```
Comment (roygbyte) [inline gameview.lua:165]: What's the reason for this `elseif` conditional? I was not aware there are issues on larger puzzles! :o

File: library.lua [hunk 3]
```diff
@@ -103,17 +103,18 @@ function Library:showDirectoryView(path_to_directory)
...
+      close_callback = function() self:onClose() end,
+   }
+   UIManager:show(LibraryPage)
 end
 
 function Library:onClose()
-	UIManager:restartKOReader()
-	--UIManager:close(self)
-	return true
+   UIManager:restartKOReader()
```
Comment (roygbyte) [inline library.lua:115]: I don't think we need to restart KOReader just to refresh the display. Your use of `Screen`'s `clear` and `refreshFull` should work?

**Pull request closed without merging**